mesa/visualization/solara_viz.py

```diff
@@ -412,6 +412,10 @@ def _check_model_params(init_func, model_params):
     """
     model_parameters = inspect.signature(init_func).parameters
     for name in model_parameters:
+        if name not in model_parameters:
```

Comment:
This check makes no sense. You are already iterating over `model_parameters` so this if will always be `False`

Reply:
oops sorry

Reply:
@quaquel do you mean something like this?
![Screenshot from 2024-11-19 01-50-43](https://github.com/user-attachments/assets/ef02d3a8-d238-4df9-8c10-c607460f7949)

Reply:
I am not intimately familiar with the `inspect` library, but this does make sense. I would advise adding a few tests to test_solara_viz.py to ensure this behaves as intended.